Tema-HTML1/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Schedule</title>
+  
+    </head>
+    <body>
+        <h1> My schedule</h1>
+        <br />
+            <label for="may2019"><i>May 2019</i></label>
+            <meter id="may2019" value="7" min="0" max="10"></meter>
+            <br /> <br />
+        <a href="http://127.0.0.1:5500/Tema-HTML1/bio.html" target="_blank">Bio</a> 
+        <a href="http://127.0.0.1:5500/Tema-HTML1/blog.html" target="_blank">Blog</a>
```

Comment:
in general, pentru a naviga prin alte pagini din interiorul aceluiasi proiect este indeajuns sa pui path-ul relativ catre fisier(ex: `/Tema-HTML1/bio.html`), fara a fi nevoie sa adaugi si [domeniul](https://www.techtarget.com/whatis/definition/domain) (`http://127.0.0.1:5500`)

```suggestion
        <a href="/Tema-HTML1/bio.html" target="_blank">Bio</a> 
        <a href="/Tema-HTML1/blog.html" target="_blank">Blog</a>
```

Reply:
Eu daca vreau sa vad fisierele in browser, fara a pornii live server, nu voi putea naviga catre acele pagini😄